ci/ansible/roles/pulp-smash/defaults/main.yml

```diff
@@ -9,3 +9,9 @@ pulp_smash_password: "admin"
 
 # If SSL certificate should be verified
 pulp_smash_verify: "false"
+
+# Pulp Server version
+# pulp_smash_version variable is optional, if you want to specify it then you
+# can do so by passing additional variables to ansible-playbook command or by
+# uncommenting the following line and editing as needed.
+# pulp_smash_version: 2.7
```

Comment:
Yes, not providing a value the if block will be skipped.

About a sane default in this case is without any value, but I still feeling that we should provide documentation about the version somewhere. The point now is, where to have that doc?

The playbook is being called on the job using `ansible-playbook -e pulp_smash_version=2.7 -e other_override_or_variable=value ...`. It will don't matter if the version variable is commented or have a sane default it will have the value specified by the value.